include/spm/evt_door.h

```diff
@@ -81,6 +81,9 @@ UNKNOWN_FUNCTION(roomBottomlesssCb);
 // evt_door_set_door_descs(DoorDesc * descs, s32 count)
 EVT_DECLARE_USER_FUNC(evt_door_set_door_descs, 1)
 
+//evt_door_set_event(char *door, int unknown, EvtScriptCode * script)
+EVT_DECLARE_USER_FUNC(evt_door_set_event, 3)
```

Comment:
The unnamed version of this function is already in the file, this should be replacing func_800e45c8